File: Steamerfy.Server/Models/Question.cs
namespace Steamerfy.Server.Models
{
    public class Question
    {
        public string QuestionText { get; set; }
        public string QuestionURL { get; set; }
        public List<List<string>> ImageURLAndOption { get; set; } = [];
        public int Answer { get; set; }
        public DateTime ExpireTime { get; set; }
        public Question(string questionText, string questionURL, List<List<string>> imageURLAndOption, int answer,int Delay)
        {
            QuestionText = questionText;
            QuestionURL = questionURL;
            ImageURLAndOption = imageURLAndOption;
            Answer = answer;
            ExpireTime = DateTime.Now.AddSeconds(Delay);
        }
    }
}

File: steamerfy.client/src/app/models/GameHub/question.ts
export class Question {
  QuestionText: string;
  QuestionURL: string;
  ImageURLAndOption: [string, string][];
  Answer: number;
  ExpireTime: Date;

  constructor(questionText: string = "", questionURL:string = "", imageURLAndOption: [string, string][] = [], answer: number = -1, expireTime: Date = new Date(0)) {
    this.QuestionText = questionText;
    this.QuestionURL = questionURL;
    this.ImageURLAndOption = imageURLAndOption;
    this.Answer = answer;
    this.ExpireTime = expireTime;
  }
}
